core/arch/arm/plat-sunxi/platform_config.h

```diff
@@ -33,25 +34,29 @@
 #include <mm/generic_ram_layout.h>
 
 /* Make stacks aligned to data cache line length */
-#define STACK_ALIGNMENT		32
+#define STACK_ALIGNMENT		64
```

Comment:
This changes the bpi_zero behavior. Is it OK?

Reply:
This should be ok as I see other 32-bit platform uses 64 for STACK_ALIGNMENT.
OR
Can we keep them separate based on size?